packaging/snap/bin/parsec-setup.sh

```diff
@@ -7,14 +7,26 @@
 export PATH=${SNAP}/sbin:${SNAP}/bin:${SNAP}/usr/bin:${PATH}
 
 # Select the python provided by snap and set the right python path
-PYTHON=${SNAP}/usr/bin/python3.9
+export PYTHON=${SNAP}/bin/python3.9
 export PYTHONPATH=${SNAP}/site-packages
 
 # We can't rely on `ctypes.util.find_library` in classic snap environment,
 # so we use the `*_LIBRARY_PATH` environment variables.
 export FUSE_LIBRARY_PATH=${SNAP}/lib/x86_64-linux-gnu/libfuse.so.2
-export SSL_LIBRARY_PATH=${SNAP}/usr/lib/x86_64-linux-gnu/libssl.so.1.1
-export CRYPTO_LIBRARY_PATH=${SNAP}/usr/lib/x86_64-linux-gnu/libcrypto.so.1.1
+# We directly use the provide ssl lib by `core20`.
+export SSL_LIBRARY_PATH=/snap/core20/current/lib/x86_64-linux-gnu/libssl.so.1.1
```

Comment:
Why not the $SNAP env var ? Doesn't it bring issues if the snap folder is not at the default location?

Reply:
The `SNAP` var only point to our snap env, so maybe using relative path could work :thinking: 

Since now we check if the lib file exist that would prevent running parsec with an invalid lib path